homepage/homepage.css

```diff
@@ -1,53 +1,100 @@
-:root{
-    --border-color:rgba(94, 91, 91, 0.493);
-    --footer-color:#242729
-}
-
-.span {
-    color: var(--main-Cta-color);
+:root {
+    --border-color: rgba(94, 91, 91, 0.493);
+    --footer-color: #242729;
+    --primary-pink-color: rgb(173, 73, 90);
+    --secondry-pink-color: rgb(247, 155, 170);
 }
 
 body {
     height: 100%;
 }
 
+/* nav bar */
+.span {
+    color: var(--main-Cta-color);
+}
+
 .search-bar input {
     width: 15rem;
     height: 2.5rem;
-    border-radius: var(--border-radius-5);
-    border: 2px solid var(--main-Cta-color);
+    border: 2px solid var(--primary-pink-color);
 }
 
 .span-color {
-    color: var(--main-Cta-color)
+    color: var(--primary-pink-color);
 }
 
 .log-in,
 .nav-link-container a {
     cursor: pointer;
-    font-weight:var(--font-bold);
+    font-weight: var(--font-bold);
     text-decoration: none;
     color: var(--text-secondry);
 }
 
 .log-in:hover,
 .nav-link-container a:hover {
-    color: var(--main-Cta-color);
-    border-bottom: 3px solid var(--main-Cta-color);
+    color: var(--primary-pink-color);
+}
+
+.right-links ul {
+    display: flex;
+}
+
+.right-links li {
+    list-style: none;
+}
+
+.right-links li a {
+    font-size: var(--size-1);
+    color: var(--text-secondary);
+    display: block;
+    text-decoration: none;
+}
+
+.right-links li a:hover {
+    opacity: 2;
+    color: var(--primary-pink-color);
+}
+
+/* hero-section */
+.img-hero {
+    width: 100%;
+    max-width: 100%;
+    height: 20rem;
+    position: relative;
+    text-align: center;
+}
+
+.overlay-text {
+    position: absolute;
+    top: 6rem;
+    left: 8rem;
+    font-size: var(--size-2);
+}
+
+.hero-btn {
+    background-color: var(--secondry-pink-color);
+    border-radius: var(--border-radius);
+}
+
+.hero-btn:hover {
+    background-color: var(--footer-color);
+    color: var(--secondry-color);
 }
 
 .hero-section {
     display: flex;
-    flex-direction: row;   
+    flex-direction: row;
```

Comment:
row is default flex-direction, you need not to specify it.

Reply:
changes made